addons/binding/org.openhab.binding.network.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name:  network.test Binding Tests
+Bundle-SymbolicName: org.openhab.binding.network.test;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Fragment-Host: org.openhab.binding.network
+Import-Package: org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.util,
+ org.hamcrest;core=split,
+ org.junit;version="4.0.0",
+ org.mockito,
+ org.openhab.binding.network,
+ org.openhab.binding.network.handler,
+ org.osgi.framework,
+ org.osgi.service.device,
+ org.slf4j
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
I'm puzzled to be honest, because I have done this in all my test bundles, and some got recently accepted to the ESH repository. And it is actually not made by me this way but by the skeleton script. Should I still change it? @martinvw